tweaks/steam.yaml

```diff
@@ -1857,6 +1854,9 @@
 "70600": # Worms Ultimate Mayhem
   compat_tool: proton_63
 
+"327030": # Worms W.M.D
```

Comment:
I need to check this game out, but from what I recall it works far better on proton, especially since I believe the native client doesn't support online cross-play?

We have to be careful with adding launch options since they can't really be removed. e.g. see Blasphemous where we had to add an empty launch option to remove the old one.

Reply:
I tried the proton version, but the problem is that there are intro videos that are not playing, and require to press Esc several times to reach the main menu. I couldn't find a specific proton version that fixes this problem, but I didn't try them all.

I didn't try to play online though.

I'll test more things and report back. 

As for the launch options remark, I don't understand how it's a problem ?

Reply:
I successfully played an unranked game against complete strangers, so at least online works. I have yet to test cross-play.

Reply:
So I have played a game online with my brother who was on Windows, so cross-play works. This was on my Arch desktop but there's no reason that it shouldn't work on ChimeraOS.

The only issue I've found is that the Steam overlay doesn't show up on desktop. It does on ChimeraOS, but then the controller doesn't work inside. I'd say it's a good tradeoff compared to the intro videos issue on the Proton version.